### ISSUE TITLE
Clean up stale/outdated issues in SCT repo

### ISSUE THREAD

**Issue by joshuacwnewton** [neuropoly/spinalcordtoolbox#2942]
We are in the process of adding priority labels to issues to help with issue triaging. However, from Slack (@jcohenadad):

> i'm realizing that the older i go, the less relevant issues are (i've closed a few of them as they became stale or irrelevant). It would save me some time if one of you could filter some of those very old and irrelevant issues, and once this is done I can go back and apply priority labels when necessary. If you are not sure about the purpose/relevance of an issue, please don't hesitate to tag me and i'll have a look (the older the issues, the less well documented they are, unfortunately)

### Progress

To view progress (i.e. for recently closed issues), please click this link: https://github.com/neuropoly/spinalcordtoolbox/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc

* **Start:** Issue 2172 (page 7 of issues) with total 14 pages.
* **End:** Issue 612 (page 11 of issues) with total 12 pages. (Last updated: October 15th)

**Comment by joshuacwnewton** [comment]
Started at issue 2172 (page 7), closed several issues, then stopped before 2054 (also page 7).

Doing this thoroughly is quite time-consuming... it's got me thinking about possibly enabling a [stale issue bot](https://probot.github.io/apps/stale/) to mitigate this in the future once the current backlog is taken care of.

**Comment by jcohenadad** [comment]
> Doing this thoroughly is quite time-consuming... it's got me thinking about possibly enabling a stale issue bot to mitigate this in the future once the current backlog is taken care of.

hum... i’m not a big fan of this bot. Many issues can still be relevant despite being inactive, and since we are a small team these inactivities do happen against our will.

**Comment by joshuacwnewton** [comment]
(Copying and pasting response from Slack for visibility)'

> hum... i’m not a big fan of this bot. Many issues can still be relevant despite being inactive, and since we are a small team these inactivities do happen against our will.

I (mostly) agree! The only thing I was thinking is to use the settings of the bot not to close the issues, but just to remind us to revisit issues in our backlog.

Right now, our current approach is to revisit the backlog in big chunks. (We've had to go through 14 total pages of old issues this time around.) But, if we get reminders X days after opening, then it will stagger and possibly be easier to manage

**Comment by joshuacwnewton** [comment]
I stopped before issue 1893 (page 8 currently) and closed a number of other issues, so now there's only 13 pages of issues. Closing the gap, slowly but surely. :)

**Comment by joshuacwnewton** [comment]
I have finished a first pass (with lots of help from @kousu)! Down to 11 pages of issues, now. :smile: 

I closed mostly issues that were quick to confirm, e.g.:
* Bugs/enhancements that involved outdated functions/scripts
* Issues that had already been addressed but were never closed
* Duplicates, or issues that were superseded by newer discussion

But, I left issues that were more challenging for me to verify quickly:
* Enhancement ideas that involve domain-specific SCT knowledge
* Bugs that require validation with specific data (i.e. not just `sct_testing_data`)

A second pass could be useful, but would take more time. Still, I think the backlog should be easier to go through for more priority labeling, @jcohenadad. So, I'm closing this. :)
